tlog-mirror.md

```diff
@@ -295,3 +297,31 @@ committed. Mirror clients will then reupload the subsequent entries.
 A mirror MAY additionally implement other update processes, provided it continues
 to correctly operate `add-entries` and never violates its cosigner requirements
 on mirror checkpoints.
+
+## Pruning and Retention Policies
+
+As with a [tiled transparency log][], a mirror maintains a *minimum index*
+value, which determines which entries it mirrors. The discussion for origin logs
```

Comment:
Maybe link to the pruning section of the tlog-tiles spec here?

Reply:
Ah good call.